config/config.go

```diff
@@ -58,8 +58,11 @@ type APIConfig struct {
 }
 
 type GRPCConfig struct {
-	ListenAddr        string `mapstructure:"listen-addr"`
-	ConnectionTimeout int64  `mapstructure:"connection-timeout"`
+	ListenAddr           string `mapstructure:"listen-addr"`
+	ConnectionTimeout    int64  `mapstructure:"connection-timeout"`
+	MaxConnectionSize    int    `mapstructure:"max-connection-size"`
```

Comment:
```suggestion
	MaxConnections    int    `mapstructure:"max-connections"`
```
Would be clearer to rename this as above, since someone may misunderstand that this is for a max number of bytes that a single connection can handle.

reference: https://github.com/tendermint/tendermint/blob/35581cf54ec436b8c37fabb43fdaa3f48339a170/config/toml.go#L185

Reply:
Ah, I got why this conversation is marked as resolved. This change had been made in the next PR.
Please ignore this comment.